modules/sdk-coin-xlm/src/xlm.ts

```diff
@@ -246,7 +259,7 @@ export class Xlm extends BaseCoin {
    * @returns is it valid?
    */
   isValidPub(pub: string): boolean {
-    return stellar.StrKey.isValidEd25519PublicKey(pub);
+    return stellar.StrKey.isValidEd25519PublicKey(pub) || isValidEd25519PublicKey(pub);
```

Comment:
why both? it looks confusing

Reply:
the stellar validation method only permits the stellar specific G-prefix version of public key, it fails to validate raw root keys in hex form, that is what the second method is there for, will add a code comment to clarify